cypress/support/commands.js

```diff
@@ -28,14 +28,74 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 
 addMatchImageSnapshotCommand();
 
-Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+Cypress.Commands.add('getByDataQa', (selector) => {
+  cy.get(`[data-qa="${selector}"]`);
 });
 
 Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
   cy.request('POST', '/users', {
     email,
     username,
     password
+  }).then(response => {
+    const user = {
+      id: response.body.user.id,
+      username: response.body.user.username,
+      email: response.body.user.email,
+      bio: response.body.user.bio,
+      image: response.body.user.image,
+      token: response.body.user.token
+    };
+    window.localStorage.setItem('user', JSON.stringify(user));
+    cy.setCookie('drash_sess', response.body.user.token);
+  });
+});
+
+Cypress.Commands.add('createArticle', (title, description, body, tags) => {
```

Comment:
The `createArticle` function creates a new user before creating an article. This may not be necessary if you have a 'login' command and the `register` command. Consider refactoring to use existing commands and avoid unnecessary requests.